test/films/films.fg

```diff
@@ -15,6 +15,9 @@ PATCH film.slug / (module.path.join films.csj) 200 {"records": 3}
 GET film.slug /iawl 200 {"title": "It's a Wonderful Life"}
 PATCH film.slug /iawl {"title": "It's a Wonder Life"} 200 {"title": "It's a Wonder Life"}
 
+# Should be able to support HEAD automatically with GET
+HEAD film.slug /iawl 200 {}
```

Comment:
That extra argument after the 200 should be an error as it can't be matched against anything

Reply:
ok, is there a check for the response should have no body yet?